result/yield-star-spacing.js

```diff
@@ -0,0 +1,24 @@
+/*eslint yield-star-spacing: ["error", "after"]*/
+/*eslint-env es6*/
+
+function* generator() {
+  yield* other();
+}
+/*eslint yield-star-spacing: ["error", "before"]*/
+/*eslint-env es6*/
+
+function *generator() {
+  yield *other();
+}
+/*eslint yield-star-spacing: ["error", "both"]*/
+/*eslint-env es6*/
+
+function * generator() {
+  yield * other();
```

Comment:
__[development][eslint]__
* Error - Unexpected space before *. (yield-star-spacing)
* Error - Unexpected space after *. (yield-star-spacing)
* Error - 'other' is not defined. (no-undef)